spec/lib/bolt_spec/project.rb

```diff
@@ -43,7 +44,7 @@ def config_path
     end
 
     def project_config
-      {}
+      { 'name' => @project_name }
```

Comment:
We probably want to update `with_project` to do something like:

```
    def with_project(name = 'project')
      Dir.mktmpdir(nil, Dir.pwd) do |tmpdir|
        @tmpdir       = Pathname.new(tmpdir).expand_path
        @project_path = (@tmpdir + name).expand_path
        @config_path  = (@project_path + 'bolt-project.yaml').expand_path

        FileUtils.mkdir_p(@project_path)
        File.write(@config_path, { 'name' => name }.merge(project_config).to_yaml)  <--- THIS
        yield
      end
    end
```

That way the project has a name even if `project_config` is overridden for a specific test.

Reply:
What if the test specifically doesn't want to include a name? It seems more clear to let devs set the entire config than to have some config magically set.